lib/Service/CardService.php

```diff
@@ -390,6 +391,29 @@ public function update($id, $title, $stackId, $type, $owner, $description = '',
 		return $card;
 	}
 
+	public function cloneCard(int $id, ?int $targetStackId = null):Card {
+		$this->permissionService->checkPermission($this->cardMapper, $id, Acl::PERMISSION_READ);
+		$originCard = $this->cardMapper->find($id);
+		if ($targetStackId == null) {
```

Comment:
```suggestion
		if ($targetStackId === null) {
```

Always do strict comparison to be safe